tests/test_app.py

```diff
@@ -40,3 +40,14 @@ def test_get_login(client, injector):
 def test_get_logout(client, injector):
     response = client.get("/saml/logout")
     assert response.status_code == 302, response.data
+
+
+def test_search_results(client, injector, mock_person_data):
```

Comment:
Line 44:
Created this new test, maybe there is a way to modify the existing search test and function instead.